cont3xt/vueapp/src/components/pages/AuditHistory.vue

```diff
@@ -0,0 +1,273 @@
+<template>
+  <div class="container-fluid">
+    <!--  search bar form  -->
+    <b-input-group class="flex-grow-1 mr-2">
+      <template #prepend>
+        <b-input-group-text>
+              <span class="fa fa-search fa-fw"/>
+        </b-input-group-text>
+      </template>
+      <b-form-input
+          tabindex="0"
+          ref="search"
+          v-model="filter"
+          debounce="400"
+          placeholder="Search history by indicator, iType, or tags"
+      />
+      <template #append>
+        <b-button
+            tabindex="0"
+            @click="clearSearchTerm"
+            :disabled="!filter"
+            title="Remove the search text">
+          <span class="fa fa-close" />
+        </b-button>
+      </template>
+    </b-input-group>
+    <!--  search bar form  -->
+
+    <!-- time range inputs -->
+    <time-range-input class="mt-1"
```

Comment:
I think the time range input could be put next to the search input to utilize the space more. I don't imagine history searches are going to be too long.

Also, I think there should be a little more margin/padding between the search area and the table (I'm sure @awick  disagrees though 😄)